### PR TITLE
travis.yml: bump golang version

### PR DESCRIPTION
**Pull request by rithujohn191**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,8 @@ sudo: required
 
 go:
   - 1.7.5
```
Comment (diegs) [inline .travis.yml]: Can we drop this one, since it's vulnerable?

File: .travis.yml [hunk 2]
```diff
@@ -4,7 +4,8 @@ sudo: required
...
-  - 1.8.3
+  - 1.8.4
```
Comment (diegs) [inline .travis.yml]: Perhaps just specify `1.8` to pick up latest?

File: .travis.yml [hunk 3]
```diff
@@ -4,7 +4,8 @@ sudo: required
...
+  - 1.9.1
```
Comment (diegs) [inline .travis.yml]: Same here

**Comment by rithujohn191** [comment]
addressed feedback

**Pull request merged**